kyo-core/shared/src/main/scala/kyo/aborts.scala

```diff
@@ -4,95 +4,90 @@ import kyo.core.*
 import scala.reflect.ClassTag
 import scala.util.control.NonFatal
 
-class Aborts[-V] extends Effect[Aborts[V]]:
-    opaque type Command[T] >: Left[V, Nothing] = Left[V, Nothing]
+// Can't use an opaque type because
+// kyo-direct tests crash the compiler.
```

Comment:
I'll create a separate issue for this. I'm not sure why the compiler crashes when some effects are marked as `opaque`. It's an important limitation we should address.

Reply:
Was there an exception you can share?